src/components/Comments.jsx

```diff
@@ -73,6 +75,13 @@ function Comments({ review_id }) {
                 comment_id={comment.comment_id}
                 originalVote={comment.votes}
               />
+              {comment.author === user.username ? (
+                <CommentDeleteButton
+                  comment={comment}
+                  commentList={commentList}
+                  setCommentList={setCommentList}
+                />
+              ) : null}
```

Comment:
At the moment, if I want to ass a comment, I have to scroll to the bottom of the comment list. Maybe rethink the layout / comment order here

Reply:
Changed the add comment section be to outside of the expandable block.  Now shows up when comments do not, also added user feedback if they add a comment.